src/bin/commands/sync.ts

```diff
@@ -0,0 +1,41 @@
+import ora from 'ora';
+import path from 'path';
+
+import { parseSchemaDtsFile } from '@/src/bin/utils/schema';
+
+export default async (positionals: string[], token: string) => {
+  const spinner = ora('Reading schema definitions').start();
+  const lastPositional = positionals[positionals.length - 1];
+  const schemasDir = lastPositional === 'sync' ? 'schemas' : lastPositional;
+
+  const schemaDtsFile = path.join(process.cwd(), schemasDir, 'index.d.ts');
+
+  let status: 'reading' | 'uploading' = 'reading';
+  try {
+    const schemaJson = await parseSchemaDtsFile(schemaDtsFile);
```

Comment:
```suggestion
    const schemaJson = await parseSchemaDefinitionFile(schemaDtsFile);
```

This name seems cleaner, IMO. But not blocking.